chrome/content/zotero/zoteroPane.js

```diff
@@ -2750,6 +2750,16 @@ var ZoteroPane = new function()
 				return;
 			}
 			
+			//Special treatment for zotero links
+			if (uri.match(/^zotero:/)) {
```

Comment:
Any reason not to make this `if (uri.match(/(^zotero\:\/\/)((open-pdf)|(select))/)) {`? Otherwise it seems like we have to generate the page twice.